modules/core/src/smithy4s/http/internals/HttpResponseCodeSchemaVisitor.scala

```diff
@@ -101,8 +101,18 @@ class HttpResponseCodeSchemaVisitor()
           case OptionalResponseCode(f) => f(value)
         }
     }
+  }
 
+  override def biject[A, B](
```

Comment:
**suggestion:** should we handle `def refine` as well?

Reply:
Might as well ^^

Reply:
done at [2dfd250](https://github.com/disneystreaming/smithy4s/pull/1377/commits/2dfd25048c65b2b4a46a90e7b692a55b69c44f4f)